res/values/strings.xml

```diff
@@ -347,13 +362,26 @@
     <!-- mailing lists -->
     <string name="mailing_list">Mailing list</string>
     <string name="ask_show_mailing_list">Show mailing list \"%1$s\"?</string>
-
+    <string name="mailing_list_profile_info">Changes on mailing list name and image apply to this device only.</string>
 
     <!-- map -->
     <string name="filter_map_on_time">Show locations in time frame</string>
     <string name="show_location_traces">Show traces</string>
     <string name="add_poi">Send point of interest</string>
 
+    <!-- chat audit log -->
+    <string name="chat_audit_log_empty_message">System messages of this chat would appear here</string>
+    <string name="chat_audit_log_title">Audit Log for %1$s</string>
+    <string name="chat_audit_log_description">This view shows only System- and Info-Messages. Useful for finding the last chat actions without scrolling through many normal messages.</string>
+
+    <!-- punycode warning / labeled links -->
+    <!-- placeholder is domain/hostname that should be trusted -->
+    <string name="open_external_url_trust_domain">Don't ask again for %1$s</string>
+    <string name="puny_code_warning_header">Suspicious link detected</string>
+    <!-- placeholder contains the hostname converted to ascii -->
+    <string name="puny_code_warning_question">Are you sure you want to visit %1$s?</string>
+    <!-- first placeholder is original hostname with special chars, second placeholder is hostname encoded in ascii -->
+    <string name="puny_code_warning_description">You followed a link which could be misrepresenting characters by using similarly looking ones from different alphabets. Following the link labelled %1$s will lead to %2$s which is normal for non-Latin characters. If you did not expect such characters this link could be harmful</string>
```

Comment:
```suggestion
    <!-- this message is shown whenever a link with non-latin characters is clicked. first placeholder is original hostname with special chars, second placeholder is hostname encoded in ascii -->
    <string name="puny_code_warning_description">You followed a link which could be misrepresenting characters by using similarly looking ones from different alphabets. Following the link labelled %1$s will lead to %2$s which is normal for non-Latin characters. If you did not expect such characters this link could be harmful.</string>
```

Reply:
I miss the "You followed a link which could be misrepresenting characters by using similarly looking ones from different alphabets.".
It helps the more confusing technical explanation bellow, because it is easier to understand I believe.